### PR TITLE
feat: Create FinancialTransactionCategory

### PR DESCRIPTION
**Pull request by DawidStuzynski**

### CODE REVIEW

**Review by Krisu93 — CHANGES_REQUESTED** [review]

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/model/FinancialTransactionCategory.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+     * Class version 0.4.0.  SerialVersionUID needs to be updated with any change.
+     */
+    @Serial
+    private static final long serialVersionUID = 100040L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Size(max = 255)
```
Comment (odzio33) [inline src/main/java/pl/byczazagroda/trackexpensesappbackend/model/FinancialTransactionCategory.java:38]: Shoud be not empty, max size 30 chars and contains only letters and digits

Comment (DawidStuzynski) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/model/FinancialTransactionCategory.java:38]: Done

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/model/FinancialTransactionCategory.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    private String name;
+
+    @Enumerated(EnumType.STRING)
+    private FinancialTransactionType type;
```
[the remark this answers is not in the document]
Comment (DawidStuzynski) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/model/FinancialTransactionCategory.java:47]: Done

**Review by odzio33 — APPROVED** [review]
Looks good for me

**Review by Krisu93 — APPROVED** [review]
OK

**Pull request merged**